client/app/states/services/resource-details/details.state.js

```diff
@@ -6,7 +6,7 @@ export function VmsDetailsState (routerHelper) {
 function getStates () {
   return {
     'services.resource-details': {
-      url: '/:vmId',
+      url: '/:serviceId/resource=:vmId',
       params: { viewType: null },
       template: '<vm-details>',
       title: N_('Resource Details')
```

Comment:
Not related to this PR but I have been slowly replacing N_ with __ wherever I find them.

Reply:
OOOOOO gooooood call :+1: